docs/flank_mixpanel_metrics.md

```diff
@@ -1,57 +1,97 @@
+- outcome 
 - configuration
-- bundle id and package name with information about the device type
-- total cost
-- cost per device type
-- test duration
-- flank version with information if tests run on corellium or firebase
 
 Every event contains a session id and project id. By ```project id``` you can find all events from every execution with specific ```project id```. By ```session id``` you can find events from specific Flank execution.
 
-## Configuration
+## Schema version [schema_version]
 
-This event contains information about the configuration executed in Flank. This event allows us to
-track information about devices and features used in Flank.
-It could be useful to check what features are most important for the community.
+This property contains version of schema, this property should be equals to this document version.
 
-Fields reflect configuration names.
+## Flank Version [flank_version]
+
+By these event's we can check how frequently users upgrade the Flank version.
+
+## Project id [project_id]
+
+This property contains id of project used in firebase.
 
 ## Bundle and package id [app_id]
 
 This event contains information about the bundle id for ios and the package id for android project. Additionally,
 contains platform type (android, ios). This event allows us to implement a ```Who uses Flank``` report with additional breakdown by ```device type```.
 
-Fields:
+## Device types [device_types]
 
-- ```app_id```
-- ```device_type```
+This property contains information about used devices types.
 
-## Total cost and cost per device type [devices_cost]
+## Shards count [shard_count]
 
-With these event's we can realize the report ```How many millions per month in spend is Flank responsible for on Firebase Test Lab? ```
+This property contains information about count of shards.
+
+## Tests result information [test]
+
+This property contains information about count of tests and count of successful, failed and flaky tests.
 
 Fields:
 
-- ```virtual_cost```
-- ```physical_cost```
-- ```total_cost```
+- ```total```
+- ```successful```
+- ```failed```
+- ```flaky```
 
-## Test duration [total_test_time]
+## Tests duration [test_duration]
 
-With these event's we can check how long tests took.
+This property contains information about time of execution tests by device type (virtual, physical, total)
 
 Fields:
 
-- ```test_duration```
+- ```virtual```
+- ```physical```
+- ```total```
 
-## Flank Version [flank_version]
+## Outcome [outcome]
 
-By these event's we can check how frequently users upgrade the Flank version.
+This property contains information about outcome of the entire run. More information you can find here: https://github.com/Flank/flank/blob/3a213579b0b8ed7ca018314f0619146408292b35/docs/feature/summary_output.md#possible-outputs
+
+## Total cost and cost per device type [cost]
+
+With these event's we can realize the report ```How many millions per month in spend is Flank responsible for on Firebase Test Lab? ```
```

Comment:
```suggestion
With these event's we can realize the report ```How many millions per month is spent on Firebase Test lab by way of using Flank? ```
```